marklogic-data-hub-central/ui/e2e/qa-project/gradle-dhs.properties

```diff
@@ -0,0 +1,23 @@
+mlDHFVersion=5.3-SNAPSHOT
+
+mlIsHostLoadBalancer=true
+mlIsProvisionedEnvironment=true
+mlStagingAuth=basic
+mlFinalAuth=basic
+mlJobAuth=basic
+mlAppServicesAuthentication=basic
+mlJobSimpleSsl=true
+mlFinalSimpleSsl=true
+mlStagingSimpleSsl=true
+mlAdminScheme=https
```

Comment:
For clarity, it's worth removing mlAdminScheme too, since it's not used (mlAdminSimpleSsl would need to be true as well if it were used). I'll approve though, it's a minor thing.

Reply:
okay sure I'll remove when I add another commit.